cmd/timoni/list.go

```diff
@@ -113,6 +100,38 @@ func runListCmd(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func listInstancesFromFlags() ([]*apiv1.Instance, error) {
+	sm, err := runtime.NewResourceManager(kubeconfigArgs)
+	if err != nil {
+		return nil, err
+	}
+
+	iStorage := runtime.NewStorageManager(sm)
+
+	ctx, cancel := context.WithTimeout(context.Background(), rootArgs.timeout)
+	defer cancel()
+
+	ns := *kubeconfigArgs.Namespace
+	if listArgs.allNamespaces {
+		ns = ""
+	}
+
+	return iStorage.List(ctx, ns, listArgs.bundleName)
+}
+
+func completeWithInstanceList(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	instances, err := listInstancesFromFlags()
+	if err != nil {
+		return nil, cobra.ShellCompDirectiveError
+	}
+	completions := make([]string, len(instances))
+	for i, inst := range instances {
+		completions[i] = fmt.Sprintf("%s\tInstalled in namespace %q",
+			inst.Name, inst.Namespace)
+	}
```

Comment:
```suggestion
	var completions []string
	for _, inst := range instances {
		if strings.HasPrefix(inst.Name, toComplete) {
			completions = append(completions, inst.Name)
		}
	}
```

I suggest we do completions like in kubectl.